src/c++/perf_analyzer/CMakeLists.txt

```diff
@@ -32,6 +32,7 @@ if(WIN32)
 else()
 
 add_subdirectory(client_backend)
+add_subdirectory(c_api_helpers)
```

Comment:
Not sure if you want to do this as a part of your changes but we should have build flags for the different types of backend i.e. local, triton (server), TFS ...
Maybe add a TODO for now or a file a ticket for the same